rllib/tuned_examples/sac/multi_agent_pendulum_sac_envrunner.py

```diff
@@ -0,0 +1,70 @@
+from ray.rllib.algorithms.sac import SACConfig
```

Comment:
Remove the `_envrunner` from the filename. These (new stack + env runner enabled) should be the new "normal".

Reply:
Can we add:
- this very test to the BUILD file? (multi-agent, local single-CPU learner)
- This might not work yet, but let's try: Add this test, but with num_gpus=2 (and tag the test in BUILD as "multi_gpu") to BUILD, so it is actually run on a >=2 GPU machine (this is the ultimate proof that these things work).
- TODO: Add single-agent, multi-GPU version of this to BUILD (I think there is also still some bug blocking this)